explorer/app/components/Project/components/common/projectMapper.ts

```diff
@@ -0,0 +1,12 @@
+// App dependencies
```

Comment:
Thank you for going ahead and splitting out the binding of the API response from the building of the view model!

Can we keep the naming convention of the binding of the API response as `xTransformer` (so in this case, `projectTransformer`) and then rename the current `projectTransformer` ([here](https://github.com/clevercanary/data-browser/blob/84-detail-description-refactor/explorer/site-config/hca-dcp/dev/projectTransformer.ts)) to be `projectViewModelBuilder`?

Reply:
Great name - thank you!